src/components/dialogs/parameters/parameters.js

```diff
@@ -265,14 +248,7 @@ export const useParametersBackend = (
                     headerId: 'fetchDefault' + type + 'ProviderError',
                 });
             });
-    }, [
-        type,
-        backendFetchDefaultProvider,
-        providers,
-        provider,
-        updateProvider,
-        snackError,
-    ]);
+    }, [type, backendFetchDefaultProvider, updateProvider, snackError]);
```

Comment:
yes, removing 'providers' is  a good point (seems to avoid multiple effects)

Reply:
Did you see why this effect was even calling `resetProvider` sometimes...
this was random, depending on two fetch performance... amazing...

Reply:
non did not see that